README.md

```diff
@@ -153,19 +95,19 @@ would look like:
 class EmailService < Polyn::Service
   event "user.added", :send_welcome_email
   event "user.updated", :send_update_email
-  
-  
+
+
```

Comment:
We must not have rubocop set up on this repo. It would complain about the double blank lines here.

Reply:
does rubocop watch markdown files?

Reply:
oh yeah, I guess that's a good point... 😅